tools/src/lib/bebopRate.ts

```diff
@@ -88,13 +88,15 @@ export const bebopRate = tool(
       input.toAsset.precision
     );
 
-    return {
+    const content = {
       buyAmountCryptoBaseUnit,
       buyAmountCryptoPrecision,
       buyTokens: data.routes[0].quote.buyTokens,
       sellTokens: data.routes[0].quote.sellTokens,
       originalData: data,
-    };
+    }
+
+    return [content, content];
```

Comment:
Tuple of `[content, artifact]`:
https://js.langchain.com/docs/how_to/tool_artifacts/#defining-the-tool